### PR TITLE
Ddf 1158

### PR DESCRIPTION
**Pull request by ricklarsen**

### CODE REVIEW

**Comment by shaundmorris** [comment]
Mostly just re-name files here? The travis build issue should have been resolved last week so your builds should be good now.

**Comment by ricklarsen** [comment]
Yeah, I was missing a line, so I put that back in. Should be working now.

File: docs/src/main/resources/Extending.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 = Extending {branding} Platform Application
-include::config.adoc[]
+include::../includes/config.adoc[]
```
Comment (shaundmorris) [inline docs/src/main/resources/Extending.adoc]: Talked to @ricklarsen but these includes have relative paths that shouldn't be there

**Comment by ricklarsen** [comment]
fixed

**Pull request merged**